packages/regl-worldview/src/Worldview.js

```diff
@@ -36,10 +35,15 @@ export type BaseProps = {|
   onMouseUp?: MouseHandler,
   onMouseMove?: MouseHandler,
   onClick?: MouseHandler,
+|};
+
+export type BaseProps = {|
   ...Dimensions,
+  ...SharedProps,
+  defaultCameraState: CameraState,
 |};
 
-export type Props = $Diff<BaseProps, Dimensions>;
+export type Props = {| ...SharedProps, defaultCameraState?: CameraState |};
```

Comment:
So defaultCameraState is optional for Props, but not for BaseProps? Not sure why these are different.

Perhaps it would help to use the `ElementConfig` type: https://flow.org/en/docs/react/types/#toc-react-elementconfig